proto/exocore/operator/v1/query.proto

```
@@ -1,20 +1,22 @@
 syntax = "proto3";
 package exocore.operator.v1;
 
+import "google/api/annotations.proto";
```

buf

import "google/api/annotations.proto": file does not exist

(COMPILE)